docs/source/whats-new.rst

```diff
@@ -241,7 +241,7 @@ New features
 
 - Add interfacing capability to read from and write to cloud object storage directly. (#216, #240)
 - Allow environmental and calibration parameters to be optionally used in calibration in place of the values stored in data file
-- Mean volume backscattering strength (MVBS) can now be computed based on actual time interval (specified in seconds) and range (specified in meters) (#54)
+- Mean volume backscattering strength (MVBS) can now be computed based on actual time interval (specified in seconds) and echo_range (specified in meters) (#54)
```

Comment:
I think we should leave this unchanged, since these comments are release notes that refer to changes that happened at a particular time in the past. The renaming of `echo` to `echo_range` is a change happening "now", so it's not retroactive